src/hearings/containers/hearing-actuals/hearing-actuals-final-confirmation/hearing-actuals-final-confirmation.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { HearingConfirmationSource } from 'src/hearings/models/hearings.enum';
```

Comment:
src

Reply:
It is the auto-import feature of Angular, I have corrected this now.